spec/factories/loans.rb

```diff
@@ -132,6 +132,12 @@
       end
     end
 
+    trait :with_transaction do
+      after(:create) do |loan|
+        create(:accounting_transaction, project_id: loan.id)
```

Comment:
it's probably useful to have several transactions -- I'd consider doing a `create_list` here, with 3 or 4 transactions just to make sure there's always several to work with of different types for each loan. If you do add more than one, you should rename to `:with_transactions`